examples/jsm/renderers/webgpu/WebGPURenderPipelines.js

```diff
@@ -808,11 +818,19 @@ const ShaderLib = {
 		layout(set = 0, binding = 3) uniform sampler mySampler;
 		layout(set = 0, binding = 4) uniform texture2D myTexture;
 
+		#pragma node_uniforms
```

Comment:
Sorry if I miss something. What's this pragma for?

Reply:
For nothing in this moment... I will replace this for NODE_UNIFORMS like this code: https://github.com/mrdoob/three.js/pull/20421#issuecomment-699906186